script.js

```diff
@@ -1,4 +1,326 @@
+    if (
+      (input == "stone" && randomComputerMove == "scissors") ||
+      (input == "reversed stone" && randomComputerMove == "paper")
+    ) {
+      myScore = myScore + 1;
+      computerScore = computerScore + 0;
+      numberOfDraws = numberOfDraws + 0;
+      totalNumberofRounds = myScore + computerScore + numberOfDraws;
+      userWinningPercentage = ((myScore / totalNumberofRounds) * 100).toFixed(
+        2
+      );
+      computerWinningPercentage = (
+        (computerScore / totalNumberofRounds) *
+        100
+      ).toFixed(2);
+      myOutputValue =
+        myOutputValue +
+        ` You won! Great job!<br><br>Your score: ${myScore} <br><br> Computer's score: ${computerScore} <br><br>Number of draws: ${numberOfDraws} <br><br> Your winning percentage: ${userWinningPercentage}% <br><br> Computer's winning percentage: ${computerWinningPercentage}%`;
+      return myOutputValue;
+    }
+    if (
+      (input == "stone" && randomComputerMove == "paper") ||
+      (input == "reversed stone" && randomComputerMove == "scissors")
+    ) {
+      myScore = myScore + 0;
+      computerScore = computerScore + 1;
+      numberOfDraws = numberOfDraws + 0;
+      totalNumberofRounds = myScore + computerScore + numberOfDraws;
+      userWinningPercentage = ((myScore / totalNumberofRounds) * 100).toFixed(
+        2
+      );
+      computerWinningPercentage = (
+        (computerScore / totalNumberofRounds) *
+        100
+      ).toFixed(2);
+      myOutputValue =
+        myOutputValue +
+        ` You lost! Try again!<br><br>Your score: ${myScore} <br><br> Computer's score: ${computerScore} <br><br>Number of draws: ${numberOfDraws} <br><br> Your winning percentage: ${userWinningPercentage}% <br><br> Computer's winning percentage: ${computerWinningPercentage}%`;
+      return myOutputValue;
+    }
+    if (
+      (input == "stone" && randomComputerMove == "stone") ||
+      (input == "reversed stone" && randomComputerMove == "stone")
+    ) {
+      myScore = myScore + 0;
+      computerScore = computerScore + 0;
+      numberOfDraws = numberOfDraws + 1;
+      totalNumberofRounds = myScore + computerScore + numberOfDraws;
+      userWinningPercentage = ((myScore / totalNumberofRounds) * 100).toFixed(
+        2
+      );
+      computerWinningPercentage = (
+        (computerScore / totalNumberofRounds) *
+        100
+      ).toFixed(2);
+      myOutputValue =
+        myOutputValue +
+        ` It is a draw! Try again!<br><br>Your score: ${myScore} <br><br> Computer's score: ${computerScore} <br><br>Number of draws: ${numberOfDraws} <br><br> Your winning percentage: ${userWinningPercentage}% <br><br> Computer's winning percentage: ${computerWinningPercentage}%`;
+      return myOutputValue;
+    } else {
+      return `You can only input: <br> "scissors" <br> "paper" <br> "stone" <br> "reversed scissors" <br> "reversed paper" or <br> "reversed stone". <br> You have input something else, please try again!`;
+    }
```

Comment:
Your code is too long, maybe you could make it into a function helper and extract the game logic.
you can combine all the condition for win into 1 "if" , same goes for lose and drew.